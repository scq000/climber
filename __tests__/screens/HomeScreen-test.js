import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../src/screens/HomeScreen';

it('Home Snapshot', async () => {
  const snap = renderer.create(<Home />).toJSON();
  await expect(snap).toMatchSnapshot();
});
