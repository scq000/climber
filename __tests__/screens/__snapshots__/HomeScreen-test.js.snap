// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
        "resizeMode": "cover",
      }
    }
  >
    <Image
      blurRadius={30}
      source={
        Object {
          "process": [Function],
        }
      }
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": undefined,
            "width": undefined,
          },
          undefined,
        ]
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "marginLeft": 26,
          "marginTop": 42,
        }
      }
    >
      <Image
        source={
          Object {
            "process": [Function],
          }
        }
        styles={
          Object {
            "borderRadius": 76,
            "width": 76,
          }
        }
      />
      <Text
        style={
          Object {
            "alignSelf": "center",
            "color": "#000",
            "flex": 1,
            "fontSize": 30,
            "fontWeight": "bold",
            "marginLeft": 16,
          }
        }
      >
        Climber
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "marginTop": 64,
        }
      }
    >
      <View>
        <RNSVGSvgView
          bbHeight={300}
          bbWidth={300}
          focusable={false}
          height={300}
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
                "borderWidth": 0,
              },
              Object {
                "flex": 0,
                "height": 300,
                "width": 300,
              },
            ]
          }
          width={300}
        >
          <RNSVGGroup>
            <RNSVGGroup
              matrix={
                Array [
                  -1.8369701987210297e-16,
                  -1,
                  1,
                  -1.8369701987210297e-16,
                  2.842170943040401e-14,
                  300,
                ]
              }
            >
              <RNSVGPath
                d="M 149.91517700393416 15.000026647931008 A 135 135 0 1 0 150 15"
                fill={0}
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeWidth",
                    "strokeLinecap",
                  ]
                }
                stroke={4292401368}
                strokeLinecap={1}
                strokeWidth={30}
              />
            </RNSVGGroup>
          </RNSVGGroup>
        </RNSVGSvgView>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 120,
              "height": 240,
              "justifyContent": "center",
              "left": 30,
              "overflow": "hidden",
              "position": "absolute",
              "top": 30,
              "width": 240,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontFamily": "Helvetica",
                  },
                  Object {
                    "color": "#FFFFFF",
                    "fontSize": 35,
                  },
                ]
              }
            >
              Steps
            </Text>
            <Text
              style={
                Array [
                  Object {
                    "fontFamily": "Helvetica",
                  },
                  Object {
                    "color": "#FFFFFF",
                    "fontSize": 50,
                    "fontWeight": "bold",
                  },
                ]
              }
            >
              12000
            </Text>
            <Text
              style={
                Array [
                  Object {
                    "fontFamily": "Helvetica",
                  },
                  Object {
                    "color": "#FFFFFF",
                    "fontSize": 16,
                  },
                ]
              }
            >
              Today
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
